### PR TITLE
Fixes issue #152 - error with creating wandb artifact

### PR DESCRIPTION
**Pull request by AshisGhosh**
## What this does
Quick fix for #152. 

Unnecessary if the string name for the datasets change. More discussion here: f80902b 

## How it was tested
Ran training with a dataset with `/` in the string. Confirmed not to error. 

Run all tests except `test_examples_3_and_2`. Passed.

## How to checkout & try?
Run training with a dataset with `/` in the string. E.g.:
```
python lerobot/scripts/train.py     policy=act     env=aloha     env.task=AlohaInsertion-v0     dataset_repo_id=lerobot/aloha_sim_insertion_human
```

### CODE REVIEW

File: lerobot/common/logger.py [hunk 1]
```diff
@@ -95,8 +99,13 @@ def save_buffer(self, buffer, identifier):
         fp = self._buffer_dir / f"{str(identifier)}.pkl"
         buffer.save(fp)
         if self._wandb:
```
Comment (aliberts) [inline lerobot/common/logger.py:101]: This should have probably already been there
```suggestion
        if self._wandb and not self._disable_wandb_artifact:
```
cc @Cadene to confirm this is what we'd want

File: lerobot/common/logger.py [hunk 2]
```diff
@@ -95,8 +99,13 @@ def save_buffer(self, buffer, identifier):
...
+            # note wandb artifact does not accept ":" or "/" in its name
             artifact = self._wandb.Artifact(
-                self._group + "-" + str(self._seed) + "-" + str(identifier),
+                self._group.replace(":", "_").replace("/", "_")
+                + "-"
+                + str(self._seed)
+                + "-"
+                + str(identifier),
```
Comment (aliberts) [inline lerobot/common/logger.py:108]: nit
```suggestion
                + f"-{self._seed}-{identifier}",
```

File: lerobot/common/logger.py [hunk 3]
```diff
@@ -82,9 +82,13 @@ def save_model(self, policy: Policy, identifier):
...
             if self._wandb and not self._disable_wandb_artifact:
-                # note wandb artifact does not accept ":" in its name
+                # note wandb artifact does not accept ":" or "/" in its name
                 artifact = self._wandb.Artifact(
-                    self._group.replace(":", "_") + "-" + str(self._seed) + "-" + str(identifier),
+                    self._group.replace(":", "_").replace("/", "_")
+                    + "-"
+                    + str(self._seed)
+                    + "-"
+                    + str(identifier),
```
Comment (aliberts) [inline lerobot/common/logger.py:91]: nit
```suggestion
                    + f"-{self._seed}-{identifier}",
```

**Review by aliberts — CHANGES_REQUESTED** [review]
Thank you! Minor suggestions, otherwise LTGM ;)

File: lerobot/common/logger.py [hunk 4]
```diff
@@ -82,9 +82,10 @@ def save_model(self, policy: Policy, identifier):
             # Also save the full Hydra config for the env configuration.
             OmegaConf.save(self._cfg, save_dir / "config.yaml")
             if self._wandb and not self._disable_wandb_artifact:
-                # note wandb artifact does not accept ":" in its name
+                # note wandb artifact does not accept ":" or "/" in its name
                 artifact = self._wandb.Artifact(
-                    self._group.replace(":", "_") + "-" + str(self._seed) + "-" + str(identifier),
+                    self._group.replace(":", "_").replace("/", "_")
+                    + f"-{self._seed}-{identifier}",
```
Comment (aliberts) [inline lerobot/common/logger.py:88]:
```suggestion
                    f"{self._group.replace(':', '_').replace('/', '_')}-{self._seed}-{identifier}",
```

File: lerobot/common/logger.py [hunk 5]
```diff
@@ -94,9 +95,11 @@ def save_buffer(self, buffer, identifier):
         self._buffer_dir.mkdir(parents=True, exist_ok=True)
         fp = self._buffer_dir / f"{str(identifier)}.pkl"
         buffer.save(fp)
-        if self._wandb:
+        if self._wandb and not self._disable_wandb_artifact:
+            # note wandb artifact does not accept ":" or "/" in its name
             artifact = self._wandb.Artifact(
-                self._group + "-" + str(self._seed) + "-" + str(identifier),
+                self._group.replace(":", "_").replace("/", "_")
+                + f"-{self._seed}-{identifier}",
```
Comment (aliberts) [inline lerobot/common/logger.py:102]:
```suggestion
                f"{self._group.replace(':', '_').replace('/', '_')}-{self._seed}-{identifier}",
```

**Review by aliberts — CHANGES_REQUESTED** [review]
Style is failing, should be good with this format (you can also make sure it will pass beforehand with `pre-commit run -a`)
Sorry for the back-and-forth ^^

**Comment by AshisGhosh** [comment]
> Style is failing, should be good with this format (you can also make sure it will pass beforehand with `pre-commit run -a`) Sorry for the back-and-forth ^^

Whoops that's on me - happy to get it right.

**Review by aliberts — APPROVED** [review]
LGTM, thanks!

**Pull request merged**